checklists/waf_checklist.en.json

```diff
@@ -4674,6 +4674,7 @@
             "link": "https://learn.microsoft.com/azure/ddos-protection/ddos-protection-overview",
             "service": "VNet",
             "severity": "High",
+            "graph": "resources | where type =~ 'microsoft.network/publicIPAddresses' | extend properties = parse_json(properties) | project id, name, location, ddosProtectionPlan = properties.ddosSettings.protectionPlan | extend compliant = iff(isnotempty(ddosProtectionPlan), 1, 0) | where compliant == 1  project id, compliant",
```

Comment:
- I believe that there is a `|` missing after `compliant == 1` right?
- But more importantly, could you please remove the `where compliant == 1` altogether? Otherwise the non-compliant resources don't show up.